oap-server/server-library/library-server/src/main/java/org/apache/skywalking/oap/server/library/server/http/HTTPServer.java

```diff
@@ -119,4 +123,14 @@ public void addHandler(Object handler, List<HttpMethod> httpMethods) {
     public void start() {
         sb.build().start().join();
     }
+
+    private String transformAbsoluteURI(final String uri) {
+        if (uri.startsWith("https://")) {
+            return uri.substring(uri.indexOf("/", 9));
```

Comment:
Why `9` of `uri.indexOf("/", 9)`? Length of `https://` is 8 only. What do I miss?

Reply:
`"https://".length=8`
`uri.indexOf("/", 9)` means that find the index of '/' from `"https://".length+1`

Reply:
OK, I should find index of next `/` begin with `"https://".lengt` rather than `"https://".lengt+1`. It's a mistake.